scrapers/urbandictionary.rb

```diff
@@ -21,6 +25,13 @@ def get_definitions(term)
     definitions.map { |m| parse_definition(term, m) }
   end
 
+  def get_random_definition()
+    uri = URL(DOMAIN+RANDOM)
```

Comment:
It is actually `URI` not `URL` (my bad). Yes we can replace how it is with your suggestion! However it should be `URL(DOMAIN).join(RANDOM)` (from my research)